bin/stylus

```diff
@@ -398,25 +398,25 @@ function help(name) {
   switch (browser) {
     case 'safari':
     case 'webkit':
-      url = 'https://developer.apple.com/library/safari/search/?q=' + name;
+      url = 'https://developer.apple.com/search/?q=' + name;
       break;
     case 'opera':
-      url = 'http://dev.opera.com/search/?term=' + name;
+      url = 'https://dev.opera.com/search/?q=' + name;
       break;
     case 'w3c':
-      url = 'http://www.google.com/search?q=site%3Awww.w3.org%2FTR+' + name;
+      url = 'https://www.google.com/search?q=site%3Awww.w3.org%2FTR+' + name;
       break;
     case 'ms':
-      url = 'http://social.msdn.microsoft.com/search/en-US/ie?query=' + name + '&refinement=59%2c61';
+      url = 'https://learn.microsoft.com/en-us/search/?terms=' + name;
       break;
     case 'caniuse':
-      url = 'http://caniuse.com/#search=' + name;
+      url = 'https://caniuse.com/#search=' + name;
```

Comment:
```diff
- 'https://caniuse.com/#search='
+ 'https://caniuse.com/?search='
```
`#` work , but maybe `?` is better ?
![2022-09-19 10 33 35](https://user-images.githubusercontent.com/14012511/190941284-8c4792fd-2ae5-43ee-ab07-ab12eda20bdd.png)